main.go

```diff
@@ -917,10 +917,12 @@ func parseResults(input *string) (map[string]struct{}, error) {
 	)
 	for _, value := range values {
 		switch value {
-		case "verified", "unknown", "unverified", "filtered_unverified":
+		case "verified", "unknown", "unverified", "filtered":
 			results[value] = struct{}{}
+		case "filtered_unverified":
```

Comment:
`filtered_unverified` is a bit wordy. I think `filtered` is better and fits in with the existing options.